travis.sh

```diff
@@ -8,7 +8,7 @@ function cleanPhpInMavenRepository {
 
 function configureTravis {
   mkdir ~/.local
-  curl -sSL https://github.com/SonarSource/travis-utils/tarball/v27 | tar zx --strip-components 1 -C ~/.local
+  curl -sSL https://github.com/SonarSource/travis-utils/tarball/v33 | tar zx --strip-components 1 -C ~/.local
```

Comment:
@vilchik-elena OK, fixed in commit https://github.com/SonarSource/sonar-php/pull/121/commits/bef7375ef92c242acddb9e1a5c22e4be82056499